project/src/events/evaluations.js

```diff
@@ -0,0 +1,6 @@
+const goToEvaluationsPage = function () {
+    const divEl = document.querySelector('#app');
```

Comment:
Consider also cleaning up event listeners with removeEventListener when you navigate from page to page

Reply:
ok, I will.